src/sidebar/components/test/Menu-test.js

```diff
@@ -183,15 +183,20 @@ describe('Menu', () => {
       shouldClose ? 'closes' : "doesn't close"
     } when user performs a "${eventType}" (key: "${key}") on menu content`, () => {
       clock = sinon.useFakeTimers();
-      const wrapper = createMenu({ defaultOpen: true });
-      wrapper.find('.Menu__content').simulate(eventType, { key });
-      // The close event is delayed by a minimal amount of time in
-      // order to allow links to say in the DOM long enough to be
-      // followed on a click. Therefore, this test must simulate
-      // time passing in order for the menu to close.
-      clock.tick(1);
-      wrapper.update();
-      assert.equal(isOpen(wrapper), !shouldClose);
+      try {
+        const wrapper = createMenu({ defaultOpen: true });
+        wrapper.find('.Menu__content').simulate(eventType, { key });
+        // The close event is delayed by a minimal amount of time in
+        // order to allow links to say in the DOM long enough to be
+        // followed on a click. Therefore, this test must simulate
+        // time passing in order for the menu to close.
+        clock.tick(1);
+        wrapper.update();
+        assert.equal(isOpen(wrapper), !shouldClose);
+      }
+      finally {
+        clock.restore();
```

Comment:
There is a module-level `afterEach` which has a `clock.restore()` call, so this should't be needed.

Reply:
When a fake clocks is used on a single test, I think it is better to declare and restore the fake clock within the function. I have removed the block-level variable. I think this is beneficial from a readability perspective.

Reply:
OK, that sounds sensible to me.